run--01a--prep-geo-data.Rmd

```diff
@@ -165,22 +165,42 @@ try(zctaShp   <-    zctas(cb = TRUE, year = tiger_year, starts_with = zcta_start
   # the zctas() function can only return by state for years 2000 and 2010 
 
 # Download school district file from Census using tigris package
-elShp <- school_districts(state = {my_state_fip}, type = 'elementary', year = tiger_year)
-scShp <- school_districts(state = {my_state_fip}, type = 'secondary',  year = tiger_year)
-unShp <- school_districts(state = {my_state_fip}, type = 'unified',    year = tiger_year)
+elShp <- school_districts(state = {my_state_fip}, type = 'elementary', year = tiger_year + 1)
+scShp <- school_districts(state = {my_state_fip}, type = 'secondary',  year = tiger_year + 1)
+unShp <- school_districts(state = {my_state_fip}, type = 'unified',    year = tiger_year + 1)
 
 # Clean data
 schoolShp <- 
   bind_rows(elShp %>% mutate(level = "elementary"), 
-            scShp %>% mutate(level = "secondary"), 
             unShp %>% mutate(level = "unified"))
-rm(elShp, scShp, unShp)
+
+schoolShp_sec_included <- 
+  bind_rows(elShp %>% mutate(level = "elementary"), 
+            scShp %>% mutate(level = "secondary"),
+            unShp %>% mutate(level = "unified"))
 
 # Display school district shapefile
 ggplot() + 
   geom_sf(data = schoolShp,
           aes(fill = level)) +
   theme_void()
+
+ggplot() +
+    geom_sf(data = schoolShp_sec_included %>% filter(level == "elementary"),
+            color = "red",   
+            fill = NA) +
+    geom_sf(data = schoolShp_sec_included %>% filter(level == "unified"),
+            color = NA, 
+            fill = "yellow",
+            alpha = 0.2) + 
+    geom_sf(data = schoolShp_sec_included %>% filter(level == "secondary"),
+            color = NA, 
+            fill = "blue",
+            alpha = 0.2) + 
+    theme_void()
+  
+# We only use elementary and unified school districts.
```

Comment:
On reflection, it sounds reasonable that we exclude secondary school districts given our focus on younger populations. However, can we add some documentation to confirm this in the comment?

Reply:
I excluded secondary school districts as secondary is an aggregation of one or more elementary school districts, so elementary feeds into secondary districts. While elementary + unified school districts are mutually exclusive, and collectively exhaustive. See below (yellow: unified, blue-filled: secondary, red line: elementary)

![image](https://github.com/user-attachments/assets/997bef84-157e-43f8-ad3e-ec5937d89b78)